### DISCUSSION TITLE
Yabai configuration for automatically stacking all chrome windows

### DISCUSSION THREAD

**Discussion by ryudice** [asmvik/yabai#1787] [Q&A] [answered]
Has anybody tried to automatically stack all chrome windows without having to use the stack layout? I would like to have the chrome windows stacked together and some other window as well in the same screen.

**Comment by adorabilis** [comment, marked as answer]
Try adding this to `.yabairc`:
```shell
yabai -m signal --add app="Google Chrome" event=window_created action="yabai -m window --stack recent && yabai -m window --focus $YABAI_WINDOW_ID"
```

**Comment by ryudice** [comment]
it worked, cant thank you enough, I always have a mess with chrome windows.

**Comment by ryudice** [comment]
seems it was too soon, it works when the window gets created but then they somehow get unstacked, still very useful but if you have any ideas it would be greatly appreaciated.
